### ISSUE TITLE
Context menu isn't visually differentiated when focused

### ISSUE THREAD

**Issue by Blackbaud-MattGregg** [blackbaud/skyux1#216]
Add visual indication when context menu receives focus.

**Comment by Blackbaud-PatrickOFriel** [comment]
In [v1.5.6](https://github.com/blackbaud/skyux/releases/tag/v1.5.6)
